src/test/java/org/folio/list/service/ListServicePostRefreshTest.java

```diff
@@ -57,6 +59,8 @@ class ListServicePostRefreshTest {
   @Mock
   private AppShutdownService appShutdownService;
 
+  private final EntityManagerFlushService entityManagerFlushService = spy(new EntityManagerFlushService(mock(EntityManager.class)));
```

Comment:
This feels pretty gross, but on the plus side: it's less magic than `@Mock` and it lets us use a real instance of the service here, instead of a mock